index.js

```diff
@@ -179,6 +181,78 @@ class ServerlessS3Sync {
       });
   }
 
+  syncMetadata() {
+    const s3Sync = this.serverless.service.custom.s3Sync;
+    const cli = this.serverless.cli;
+    if (!Array.isArray(s3Sync)) {
+      cli.consoleLog(`${messagePrefix}${chalk.red('No configuration found')}`)
+      return Promise.resolve();
+    }
+    cli.consoleLog(`${messagePrefix}${chalk.yellow('Syncing metadata...')}`);
+    const servicePath = this.servicePath;
+    const promises = s3Sync.map( async (s) => {
+      let bucketPrefix = '';
+      if (s.hasOwnProperty('bucketPrefix')) {
+        bucketPrefix = s.bucketPrefix;
+      }
+      let acl = 'private';
```

Comment:
> For example, if you call the s3api copy-object on a public-read acl s3 file, and you don't explicitly give it an acl, it will make it private by default. 

Maybe, `acl` declared but not used.

Reply:
fixed this